readthedocs/settings/base.py

```diff
@@ -96,6 +96,8 @@ class CommunityBaseSettings(Settings):
     # override classes
     CLASS_OVERRIDES = {}
 
+    DOC_PATH_PREFIX = '_/'
```

Comment:
Aren't we prefixing Read the Docs settings with `RTD_`?

Reply:
That is a setting used in .com

Reply:
Yeah. I didn't want to add the prefix here too keep consistency with .com.